src/charts/utilization-bar/utilization-bar-chart.directive.js

```diff
@@ -107,6 +110,18 @@
       'type': 'inline'
     };
 
+    $scope.title6 = 'Utilization by storage type';
+    $scope.data6 = {
+      'total': '100',
+      'subdata' : [ { "used" : 45 , "color" : "#00558a" , "subtitle" : "Object" },
+                    { "used" : 15 , "color" : "#0071a6" , "subtitle" : "Block" },
+                    { "used" :  5 , "color" : "#00a8e1" , "subtitle" : "OpenStack" }]
```

Comment:
Please choose colors from the Patternfly color palette - [Color Palette](https://www.patternfly.org/styles/color-palette/)